logging/src/main/java/tech/pegasys/teku/logging/EventLogger.java

```diff
@@ -33,11 +38,27 @@ private EventLogger(final String name) {
     this.log = LogManager.getLogger(name);
   }
 
-  public void genesisEvent(final Bytes32 hashTreeRoot, final Bytes32 genesisBlockRoot) {
+  public void genesisEvent(
+      final Bytes32 hashTreeRoot, final Bytes32 genesisBlockRoot, final UnsignedLong genesisTime) {
+    final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+
+    final ZoneId systemDefaultZoneId = ZoneId.systemDefault();
```

Comment:
Wonder if we should use GMT for easier comparison ...

Reply:
Yeah, I'd probably also stick to a single timezone. GMT sounds the most widely used

Reply:
Done.